doc/content/gateways/moko-mkgw2-lw/_index.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "MOKOSMART MKGW2-LW LoRaWAN Gateway"
+description: ""
+---
+
+MOKO 8 channels gateway MKGW2 is based on LoRaWAN protocol, which is embedded with Semtech’s high performance multi-channel transceiver SX1301/SX1257 and MTK platform. It is for
+indoor use and easy for installation. 
+
+{{< figure src="mkgw2-lw.jpg" alt="mkgw2-lw" class="plain float" >}}
+
+## Requirements
+
+1. User account on {{% tts %}} with rights to create gateways.
+2. MOKOSMART MKGW2-LW connected to the internet via ethernet or WIFI.（For the ethernet and wifi setting, you can refer to the  5.login into the Gateway and 6.Internet Setting of the user maunal)
+
+## Registration
+
+Create a gateway by following [the instructions]({{< ref "/gateways/adding-gateways" >}}). 
+
+The **EUI** of the gateway can be found on the configuration web page of the gateway. 
+
+{{< figure src="eui.png" alt="Gateway EUI" >}}
+
+## Configuration via Browser
+1. Turn On the gateway and waiting for about 60s.
+2. Using your PC or phone connect the SSID of the gateway. The default SSID format is "MKGW2-LW-xxxx” such as " MKGW2-LW-91D8 ", “91D8” is the last two bytes of the gateway MAC address. verify the password (Default: Moko4321) and connect to the gateway.
```

Comment:
```suggestion
2. Using your PC or phone, enter the URL of the gateway in a browser. The default URL format is "MKGW2-LW-xxxx”, e.g "MKGW2-LW-91D8 ", where “91D8” is the last two bytes of the gateway MAC address. The default password is Moko4321.
```

This does not look like a valid url. Are you sure it's just "http://MKGW2-LW-xxxx" ?